lib/options.dart

```diff
@@ -4,39 +4,65 @@ import 'dart:ui';
 import 'package:image/image.dart' hide Color;
 import 'package:iproov_flutter/enums.dart';
 
+// TODO: For all the classes in this file, identify which properties need to be null
+// and which ones don't. Non-nullable should be the default.
+// Note: non-nullable variables need to be required or have a default value in the constructor.
+
 class Options {
-  UiOptions ui = new UiOptions();
-  NetworkOptions network = new NetworkOptions();
-  CaptureOptions capture = new CaptureOptions();
+  const Options({
+    this.ui = const UiOptions(),
+    this.network = const NetworkOptions(),
+    this.capture = const CaptureOptions(),
+  });
+  final UiOptions ui;
+  final NetworkOptions network;
+  final CaptureOptions capture;
 
   Map<String, dynamic> toJson() => {
-    'ui': ui.toJson(),
-    'network': network.toJson(),
-    'capture': capture.toJson()
-  };
+        'ui': ui.toJson(),
+        'network': network.toJson(),
+        'capture': capture.toJson()
+      };
 }
 
 class UiOptions {
-  GenuinePresenceAssuranceUiOptions genuinePresenceAssurance = new GenuinePresenceAssuranceUiOptions();
-  LivenessAssuranceUiOptions livenessAssurance = new LivenessAssuranceUiOptions();
-  Filter? filter;
-  Color? lineColor;
-  Color? backgroundColor;
-  String? title;
-  String? fontPath; // TODO: Not cross-platform
-  String? fontResource; // TODO: Not cross-platform
-  String? font;  // TODO: Not cross-platform
-  Image? logoImage;
-  Image? closeButtonImage; // TODO: Not yet supported in iOS SDK
-  Color? closeButtonTintColor;
-
-  // Drawable logoImageDrawable = null;
-  bool? enableScreenshots;
-  Orientation? orientation;
-  int? activityCompatibilityRequestCode;
+  const UiOptions({
+    this.genuinePresenceAssurance = const GenuinePresenceAssuranceUiOptions(),
+    this.livenessAssurance = const LivenessAssuranceUiOptions(),
```

Comment:
Note how non-nullable properties can have default values.

The same can be done with all the properies below if desired.